addons/cinematicSpec/functions/fn_cam.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+
+BIS_DEBUG_CAM_LASTPOS = getPos ACE_spectator_camera;
```

Comment:
in 3.10.2 : ACE_spectator_freeCamera , ACE_spectator_unitCamera

(also, use a macro to get ACE  spectator vars)